functests/4_latency/latency.go

```diff
@@ -249,22 +251,36 @@ var _ = Describe("[performance] Latency Test", func() {
 
 func getLatencyTestRun() (bool, error) {
 	if latencyTestRunEnv, ok := os.LookupEnv("LATENCY_TEST_RUN"); ok {
-		return strconv.ParseBool(latencyTestRunEnv)
+		if val, err := strconv.ParseBool(latencyTestRunEnv); err != nil {
+			return val, fmt.Errorf("the environment variable LATENCY_TEST_RUN has incorrect value %q: %w", latencyTestRunEnv, err)
+		} else {
```

Comment:
You don't need the else here, just return.

Reply:
my bad , thanks !